### PR TITLE
Fix more sources

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/data/abomination-vaults-bestiary.db/bone-gladiator.json [hunk 1]
```diff
@@ -328,7 +328,7 @@
             "privateNotes": "",
             "publicNotes": "",
             "source": {
-                "value": "Pathfinder Abomination Vaults Hardcover Compilation"
+                "value": "Pathfinder Abomination Vaults Compilation Hardcover"
```
Comment (SpartanCPA) [inline packs/data/abomination-vaults-bestiary.db/bone-gladiator.json:331]: @silvative Do you have a preference?

Comment (simonward) [reply packs/data/abomination-vaults-bestiary.db/bone-gladiator.json:331]: It's a good question, since there's one instance of each.

Comment (TikaelSol) [reply packs/data/abomination-vaults-bestiary.db/bone-gladiator.json:331]: I think hardcover compilation sounds more sensible.

Comment (simonward) [reply packs/data/abomination-vaults-bestiary.db/bone-gladiator.json]: Agreed. Fixed.

**Pull request merged**